alembic/versions/2a42cfa713b1_add_url_field.py

```diff
@@ -0,0 +1,32 @@
+"""add url field
+
+Revision ID: 2a42cfa713b1
+Revises: b22426f9e117
+Create Date: 2025-08-24 23:58:18.221319
+
+"""
+from typing import Sequence, Union
+
+from alembic import op
+import sqlalchemy as sa
+
+
+# revision identifiers, used by Alembic.
+revision: str = '2a42cfa713b1'
+down_revision: Union[str, Sequence[str], None] = 'b22426f9e117'
+branch_labels: Union[str, Sequence[str], None] = None
+depends_on: Union[str, Sequence[str], None] = None
+
+
+def upgrade() -> None:
+    """Upgrade schema."""
+    # ### commands auto generated by Alembic - please adjust! ###
+    op.add_column('book', sa.Column('img_url', sa.String(), nullable=True))
+    # ### end Alembic commands ###
+
+
+def downgrade() -> None:
```

Comment:
when would this be used?

Reply:
It allows you to undo migration steps if they are causing problems.